packages/component-submission/client/pages/FilesStepPage.js

```diff
@@ -76,8 +74,7 @@ export class FilesStepPageComponent extends React.Component {
           manuscriptUploadingError: error,
         })
       })
-      .finally(() => {
-        disableSave(false)
+      .then(() => {
```

Comment:
We'll be stuck in a state of `isUploading` if this throws an error. This will then cause the client to be stuck in the converting stater so the error will be hidden